craft.lic

```diff
@@ -344,7 +344,7 @@ class Craft
     elsif rank <= 300 # Tier 5 - Basic
       # Buy an abolition and induction sigils and bone totem
```

Comment:
Need to change `induction sigil-scroll` to `congruence sigil-scroll`

Reply:
Commited, thanks.